### ISSUE TITLE
ES6 let used outside strict mode

### ISSUE THREAD

**Issue by benadamstyles** [gradecam/jasmine-co#2]
On Node `4.4.0`, getting the following error:

```
/Users/ben/Dev/Git/.../node_modules/jasmine-co/index.js:66
                let result = userFn.call(this);
                ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
```

**Comment by putermancer** [comment]
Thanks, I've done a bit too much TypeScript lately and thought I'd caught all these. Fix published as `1.2.1`.

**Comment by benadamstyles** [comment]
Brilliant, thanks for such a quick response!
